src/components/BusinessesList/index.jsx

```diff
@@ -8,8 +8,12 @@ export default function BusinessesList() {
 
   useEffect(() => {
     (async function getBuisinesses() {
-      const { data } = await http.get(endPointUrl);
-      setBusinesses(data.businesses);
+      try {
+        const { data } = await axios.get(endPointUrl);
+        setBusinesses(data.businesses);
+      } catch (error) {
+        throw new Error(error);
```

Comment:
you are crashing the frontend again ! :laughing:

Reply:
i dont know whats going on lol